### PR TITLE
Bringing NFTs support

### PR DESCRIPTION
**Pull request by aaitor**
## Description

Using contracts `v1` version to support end to end NFTs functionality

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [X] Follows the code style of this project.
- [X] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

File: tests/nevermined/test_nfts.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    #
+    # keeper.nft_access_template.create_agreement(
+    #     nft_access_agreement_id,
+    #     asset_id,
+    #     [nft_holder_cond_id, nft_access_cond_id],
+    #     nft_access_service_agreement.conditions_timelocks,
+    #     nft_access_service_agreement.conditions_timeouts,
+    #     consumer_account.address,
+    #     pub_acc
+    # )
```
Comment (r-marques) [inline tests/nevermined/test_nfts.py:97]: Maybe we can remove this

**Review by r-marques — APPROVED** [review]
lgtm.

I think we should create a new issue to implement an API specific to the NFTs, similar to the sdk-js. This would make it easier to create, mint, burn and get balance of NFTs without having to go through the assets API

File: tests/nevermined/test_nfts.py (hunk 1 above)
Comment (aaitor) [reply tests/nevermined/test_nfts.py:97]: :+1:

**Comment by aaitor** [comment]
> lgtm.
> 
> I think we should create a new issue to implement an API specific to the NFTs, similar to the sdk-js. This would make it easier to create, mint, burn and get balance of NFTs without having to go through the assets API

Moved to new `nfts` api

**Pull request merged**